demo/demo.R

```diff
@@ -339,6 +348,16 @@ print(OutputCollect)
 ################################################################
 #### Step 4: Select and save the any model
 
+### Using lares >= 5.2.4, you can try this to help you pick a model:
+### (Update to stable or dev version using lares::updateLares())
+# lares::robyn_modelselector(
+#   InputCollect,
+#   OutputCollect,
+#   metrics = c("rsq_train", "performance", "potential_improvement",
+#               "non_zeroes", "incluster_models"),
+#   wt = c(3, 1, 1, 1, 0.1)
+# )
+
```

Comment:
Please don't include model selection recommender yet until the concept is 100% aligned